src/rules/cognitive-complexity.ts

```diff
@@ -61,13 +62,13 @@ const rule: TSESLint.RuleModule<string, (number | 'metric' | 'sonar-runtime')[]>
     type: 'suggestion',
     docs: {
       description: 'Cognitive Complexity of functions should not be too high',
-      recommended: 'error',
```

Comment:
omitting this value, implies it's not recommended to be turned on. And also, based on this value, we compile our plugin.